data_structures/src/transaction.rs

```diff
@@ -324,11 +335,28 @@ impl ScriptTransactionBody {
     /// Script transaction weight. It is calculated as:
     ///
     /// ```text
-    /// ST_weight = N*INPUT_SIZE + M*OUTPUT_SIZE*gamma
+    /// ST_weight = N*INPUT_SIZE + M*OUTPUT_SIZE*gamma + REDEEM_SCRIPTS_SIZE
```

Comment:
Should spent the same weight a VT output and a ScriptOutput?

Reply:
I assumed they can be the same because they are roughly the same size, the only difference is that the `ScriptOutput` does not have a timelock field, but that's only 8 bytes so I believe it can be ignored.